src/stamp/modeling/train.py

```diff
@@ -33,6 +33,9 @@
     EncodedTargets,
     LitVisionTransformer,
 )
+from stamp.modeling.lightning_cobra import (
+    LitCobra,
+)
```

Comment:
This will throw a ModuleNotFoundError if the `[cobra]` option isn't installed...